src/app/views/realms/view/RouteRealmsView.vue

```diff
@@ -16,80 +14,39 @@ import AppShareCard from '../../../../_common/share/card/AppShareCard.vue';
 import { ShareModal } from '../../../../_common/share/card/_modal/modal.service';
 import AppSpacer from '../../../../_common/spacer/AppSpacer.vue';
 import AppTranslate from '../../../../_common/translate/AppTranslate.vue';
-import { $gettextInterpolate } from '../../../../_common/translate/translate.service';
 import { User } from '../../../../_common/user/user.model';
-import { ActivityFeedService } from '../../../components/activity/feed/feed-service';
-import { ActivityFeedView } from '../../../components/activity/feed/view';
-import { AppActivityFeedLazy } from '../../../components/lazy';
 import AppPageContainer from '../../../components/page-container/AppPageContainer.vue';
-import AppPostAddButton from '../../../components/post/add-button/AppPostAddButton.vue';
 import AppShellPageBackdrop from '../../../components/shell/AppShellPageBackdrop.vue';
 import AppUserKnownFollowers from '../../../components/user/known-followers/AppUserKnownFollowers.vue';
+import { createRealmRouteStore, RealmRouteStoreKey } from './view.store';
 
 export default {
 	...defineAppRouteOptions({
-		resolver: async ({ route }) =>
-			Promise.all([
-				Api.sendRequest('/web/realms/' + route.params.path),
-				Api.sendRequest(
-					ActivityFeedService.makeFeedUrl(
-						route,
-						`/web/posts/fetch/realm/${route.params.path}`
-					)
-				),
-			]),
+		resolver: async ({ route }) => Api.sendRequest('/web/realms/' + route.params.path),
 	}),
 };
 </script>
 
 <script lang="ts" setup>
 const router = useRouter();
-const route = useRoute();
 
-const realm = ref<Realm>();
+const routeStore = createRealmRouteStore();
```

Comment:
destructure from this after setting it up, it's just nicer to call `realm` below instead of `routeStore.realm` each place